python/vision_explanation_methods/version.py

```diff
@@ -3,5 +3,5 @@
 name = 'vision_explanation_methods'
 _major = '0'
 _minor = '0'
-_patch = '5'
+_patch = '6'
```

Comment:
wasn't a version update already done in the last commit to this repo here:
https://github.com/microsoft/vision-explanation-methods/pull/27
it looks like that one hasn't been released yet?